enterprise/backend/test/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions_test.clj

```diff
@@ -537,61 +535,77 @@
                     {:order-by [[:asc $id]]
                      :limit 2})))))))))
 
+(defn- correct-metadata-test-col-fns []
```

Comment:
Moved the fns below out so I could split a big test with four logically separate test cases into four separate tests. I had just one of the cases failing and debugging just one was super annoying. Logically separate test cases belong in their own `deftest` form!!!!